src/codegen/wasm.zig

```diff
@@ -800,8 +817,11 @@ pub const Context = struct {
         const air_tags = self.air.instructions.items(.tag);
         return switch (air_tags[inst]) {
             .add => self.airBinOp(inst, .add),
+            .addwrap => self.airBinOp(inst, .add),
```

Comment:
I don't think this will scale for all integers (e.g. other than `i32`). This program compiled with this branch will output a wrong number:

```zig
pub export fn _start() u8 {
    var i: u8 = 255;
    return i +% 1;
}
```

I think a different function for adding and then masking bits out is necessary.

Reply:
Good call, @joachimschmidt557. I hadn't even thought of this, will be updating this PR soon with the improvements.

Reply:
Note that for non wrapping operations, larger integers can he used. For example for non wrapping i13, you can just use i32 operations. It's the wrapping operations that need to possibly do more calculations to honor twos complement wraparound.